Source/Tools/FEXQonfig/main.qml

```diff
@@ -0,0 +1,669 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Dialogs 1.3
+import QtQuick.Layouts 1.15
+
+import FEX.ConfigModel 1.0
+import FEX.EnvVarModel 1.0
+import FEX.RootFSModel 1.0
+
+ApplicationWindow {
+    id: root
+
+    visible: true
+    width: 640
+    height: 600
+    title: configDirty ? qsTr("FEX configuration *") : qsTr("FEX configuration")
+
+    property url configFilename
+
+    property bool configDirty: false // TODO: Consistently set to true on modifications
+    property bool closeConfirmed: false
+
+    signal selectedConfigFile(name: url)
+    signal triggeredSave(name: url)
+
+    // Property used to force reloading any elements that read ConfigModel
+    property bool refreshCache: false
+
+    function refreshUI() {
+        refreshCache = !refreshCache
+    }
+
+    FileDialog {
+        id: openFileDialog
+        title: qsTr("Open FEX configuration")
+        nameFilters: [ "Config files(*.json)", "All files(*)" ]
+        // TODO: Set default folder to ~/.fex-emu?
```

Comment:
Or even better, `~/.config/fex-emu/`

Reply:
You're not wrong. I'm not actually sure why we aren't yet using `XDG_CONFIG_HOME` for the config.

Reply:
^ currently FEX uses it if it's explicitly set, but defaults to ~/.fex-emu